src/pyhf/infer/test_statistics.py

```diff
@@ -47,16 +89,88 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         raise UnspecifiedPOI(
             'No POI is defined. A POI is required for profile likelihood based test statistics.'
         )
+    if par_bounds[pdf.config.poi_index][0] == 0:
+        log.warning(
+            'qmu test statistic used for fit configuration with POI bounded at zero. Use qmutilde.'
```

Comment:
```suggestion
            'qmu test statistic used for fit configuration with POI bounded at zero.\n'
            + 'Use the qmu_tilde test statistic (pyhf.infer.test_statistics.qmu_tilde) instead.'
```

Maybe something more explicit(?) and also need to match the naming convention (underscore missing).